src/main/java/com/sprint/mission/discodeit/service/MessageService.java

```diff
@@ -0,0 +1,25 @@
+package com.sprint.mission.discodeit.service;
+
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.entity.User;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface MessageService {
+    public void creat(Message message);
+
+    public void delete(UUID messageId);
+
+    public void update(UUID messageId, String updateMessage);
+
+    public List<Message> write(UUID userId, UUID channelId);
```

Comment:
인터페이스는 따로 접근제어자를 적어줄 필요는 없습니다.  
왜그런지 혹시 종우님은 아시나요?  

```suggestion
    List<Message> write(UUID userId, UUID channelId);
```